internal/app/service/service.go

```diff
@@ -141,6 +145,11 @@ func (profileSvc *service) ListProfiles(ctx context.Context) (values []specs.Res
 			isActive = "YES"
 		}
 
+		isProfileCompleteStr := "NO"
```

Comment:
Create new function in helper for isCurrentEmployee, isActive, isProfileCompleteStr

```
func checkBooleanStatus(value int) string {
    if value == 1 {
        return "YES"
    }
    return "NO"
}
```